packages/components/src/form-token-field/index.js

```diff
@@ -583,9 +583,12 @@ class FormTokenField extends Component {
 						/>
 					) }
 				</div>
-				<div id={ `components-form-token-suggestions-howto-${ instanceId }` } className="screen-reader-text">
-					{ __( 'Separate with commas' ) }
-				</div>
+				<p id={ `components-form-token-suggestions-howto-${ instanceId }` } className="components-form-token-field__help">
```

Comment:
It seems like the id `components-form-token-suggestions-howto-${ instanceId }` is not being referenced anywhere, can we remove the id?

Reply:
Referenced here https://github.com/WordPress/gutenberg/blob/master/packages/components/src/form-token-field/token-input.js#L50